src/DO/Sara/ImageProcessing/Deriche.hpp

```diff
@@ -228,24 +227,18 @@ namespace DO { namespace Sara {
     return J;
   }
 
-  //! \brief Helper class to use: Image<T,N>::compute<DericheBlur>(T sigma)
-  template <typename T, int N>
+  //! @brief Wrapper class to use: Image<T,N>::compute<DericheBlur>(T sigma)
   struct DericheBlur
```

Comment:
Image processing functor classes are not templated anymore and this is actually better.